src/Cake.Issues.PullRequests/IReportIssuesToPullRequestSettings.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace Cake.Issues.PullRequests
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Interface for Settings affecting how issues are reported to pull requests.
+    /// </summary>
+    public interface IReportIssuesToPullRequestSettings : IRepositorySettings
+    {
+        /// <summary>
+        /// Gets or sets the hash of the commit for which the issues were reported.
+        /// </summary>
+        string CommitId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the global number of issues which should be posted at maximum over all
+        /// <see cref="IIssueProvider"/>.
+        /// Issues are filtered by <see cref="IIssue.Priority"/> and issues with an <see cref="IIssue.AffectedFileRelativePath"/>
+        /// are prioritized.
+        /// Set to <c>null</c> to not set a global limit.
+        /// Default is to not set a global limit.
+        /// Use <see cref="MaxIssuesToPostForEachIssueProvider"/> to set the limit for each issue provider
+        /// and <see cref="MaxIssuesToPostAcrossRuns"/> to set a limit across multiple runs.
+        /// </summary>
+        int? MaxIssuesToPost { get; set; }
+
+        /// <summary>
+        /// Gets or sets the global number of issues which should be posted at maximum over all
+        /// <see cref="IIssueProvider"/> and across multiple runs.
+        /// Issues are filtered by <see cref="IIssue.Priority"/> and issues with an <see cref="IIssue.AffectedFileRelativePath"/>
+        /// are prioritized.
+        /// Set to <c>null</c> to not set a limit across multiple runs.
+        /// Default is to not set a limit across multiple runs.
```

Comment:
This two lines sound a littble bit strange. to ... to ... to. Maybe this could be written better, but it would also be ok for me to leave it. What about this?
``` 
/// Default is <c>null</c> which won't set a global limit across multiple runs`
```